examples/demos/elation/util/slotStyler.js

```diff
@@ -0,0 +1,26 @@
+import { getAvailabilities, getAllAvailabilities } from '../data/availabilities';
+
+const availableSlotStyle = {
+  style: { background: '#FFF' },
+};
+
+const initialPhysicianAvailabilities = Object.values(getAvailabilities(28716));
+const allAvailabilities = Object.values(getAllAvailabilities()).flat();
+
+export default function slotStyler(date, entityKey, isMultiGrid) {
+  const availabilities = isMultiGrid ? allAvailabilities : initialPhysicianAvailabilities;
+
+  const hasAvailability = availabilities.some(availability => {
```

Comment:
No-op:

Might be worth thinking about baking this directly into this calendar, specifically the understanding of which time slots have availabilities. But I don't think it's important and it can definitely come later, if it comes at all.